src/contracts/Model/Suggestion/ParentLocation.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Search\Model\Suggestion;
+
+final class ParentLocation
+{
+    private int $id;
```

Comment:
In my opinion you don't really need to keep Content object here. It can be fetched from Location if needed.